app/Http/Controllers/ApiController.php

```diff
@@ -53,6 +53,7 @@ public function update(Request $request)
             'mac_address'   => 'nullable|string|min:12|max:12',
             'time'          => 'nullable|date',
             'cover_status'  => 'nullable|string|max:32',
+            'cover_command'  => 'nullable|alpha|max:6|in:open,close,lock,unlock',
```

Comment:
Is there any use case where i would override the server with lock, unlock?

Reply:
I don't think so I was just putting in all possible values that are valid. If you have no use I will change it to close and open only.